spec/templater_spec.rb

```diff
@@ -71,7 +62,7 @@ def expect_renders(template_parameters, expected, config: Metalware::Config.new)
         alces.index: 0
         EOF
 
-        expect_renders({}, expected, config: @config)
+        expect_renders({}, expected)
```

Comment:
If you look in `expect_renders` you'll see `filesystem.test`; this runs `FileSystem.test` but using the `FileSystemConfigurator` built up in in the `let :filesystem` block above. See https://github.com/alces-software/metalware/blob/175a29421a732445feaaf14adbf1f9b2896d32a9/spec/filesystem.rb#L154-L156 for its source.